### PR TITLE
Make language feature status 'Developers' column singular

### PR DESCRIPTION
**Pull request by jnm2**
'Reviewer' and 'LDM Champ' are not plural and they contain multiple entries in a single cell.

### CODE REVIEW

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -98,15 +98,15 @@ efforts behind them.
...
 | [`ref` and `this` ordering in ref extension](https://github.com/dotnet/roslyn/pull/23643) | - | Merged | [alrz](https://github.com/alrz) | - | - |
 | [Tiebreaker for by-val and `in` overloads](https://github.com/dotnet/roslyn/pull/23122) | main | Merged | [OmarTawfik](https://github.com/OmarTawfik) | - | - |
 
 # C# 7.2
 
-| Feature | Branch | State | Developers | Reviewer | LDM Champ |
-| ------- | ------ | ----- | ---------- | -------- | --------- |
+| Feature | Branch | State | Developer | Reviewer | LDM Champ |
+| ------- | ------ | ----- | --------- | -------- | --------- |
 | [ref readonly](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.2/readonly-ref.md) | main | Merged | [vsadov](https://github.com/vsadov), [OmarTawfik](https://github.com/OmarTawfik) | [cston](https://github.com/cston),[gafter](https://github.com/gafter) | [jaredpar](https://github.com/jaredpar) |
```
Comment (Youssef1313) [inline docs/Language Feature Status.md:110]: There are two developers here.

Comment (jnm2) [reply docs/Language Feature Status.md:110]: The same is true of the Reviewer and LDM Champ columns.

Comment (Youssef1313) [reply docs/Language Feature Status.md:110]: Got it. Thanks for clarifying.

**Comment by jcouv** [comment]
Merging with single review since doc-only change.

**Comment by jcouv** [comment]
Merged. Thanks

**Pull request merged**